### PR TITLE
add getting login_user_id

### PR DESCRIPTION
**Pull request by huangaszaq**

### CODE REVIEW

File: src/wechaty_puppet_hostie/puppet.py [hunk 1]
```diff
@@ -537,7 +537,7 @@ def self_id(self) -> str:
             contact_id
         :return:
         """
-        return ''
+        return self.login_user_id
```
Comment (wj-Mcat) [inline src/wechaty_puppet_hostie/puppet.py:540]: `login_user_id` is Optional str, we should check the null status before return it. 
```python
if not self.login_user_id:
    raise ValueError('can"t call self_id() before logined')
return self.login_user_id
```

Comment (huangaszaq) [reply src/wechaty_puppet_hostie/puppet.py:540]: Get it. login_user_id is initialized as None and the function needs type of str as return.

**Comment by huangaszaq** [comment]
This pr requires python-wechaty-puppet.

**Comment by wj-Mcat** [comment]
Please fix the small issue for `contact_avatar` function. There is also a improvement for `FileBox` module : `from_json`.  You can change the `contact_avatar` final FileBox initilization code:
```python
return FileBox.from_base64(response.filebox)
```

**Comment by huangaszaq** [comment]
@wj-Mcat OK，i will fix it when the version of puppet is updated to newer.

**Comment by huangaszaq** [comment]
Or you can edit this file directly. Please feel free.

**Comment by wj-Mcat** [comment]
@huangaszaq  It has been updated 🤘 .

**Pull request merged**